include/nn/os.h

```diff
@@ -46,17 +46,17 @@ struct LightEventType {
 
 // https://github.com/misson20000/nn-types/blob/master/nn_os.h
```

Comment:
This comment can probably be removed now that the definition of EventType isn't based on that header anymore

Reply:
Fixed!